.gitattributes

```diff
@@ -0,0 +1 @@
+*.go text eol=lf
```

Comment:
It has to do with the go-linter, it will error if there is CRLF any where. When the windows runner clones the repo, it converts stuff to CRLF and the linter fails .... I think =)

Reply:
I tried setting core.autocrlf=true and got lint failure

Reply:
Ah gotcha. I think if the runner applied `git config --global core.autocrlf input` locally this wouldn't be necessary, but I don't see how that can be done.

This works just fine!